tests/ui/perfherder/modal_perf_tags_test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { render, fireEvent, waitFor } from '@testing-library/react';
+
+import TagsModal from '../../../ui/perfherder/alerts/TagsModal';
+import testPerformanceTags from '../mock/performance_tags';
+import testAlertSummaries from '../mock/alert_summaries';
+
+const testAlertSummary = testAlertSummaries[0];
+
+const testTagsModal = () => {
+  return render(
+    <TagsModal
+      alertSummary={testAlertSummary}
+      performanceTags={testPerformanceTags}
+      showModal
+      toggle={() => {}}
+      updateAndClose={() => {}}
+    />,
+  );
+};
+
+test('When the modal for performance tags is open, a list of 4 tags should be displayed', async () => {
+  const { queryAllByTestId } = testTagsModal();
+
+  const list = await waitFor(() => queryAllByTestId(/modal-perf-tag/));
+
+  expect(list).toHaveLength(4);
```

Comment:
We should use something more dynamic than the hard coded `4`.
Like computing the expected list size from the fixture itself.

Reply:
We still need to address this.